core/controllers/practice_sessions.py

```diff
@@ -52,16 +54,22 @@ def get(self, topic_name):
         # Topic cannot be None as an exception will be thrown from its decorator
         # if so.
         topic = topic_fetchers.get_topic_by_name(topic_name)
+        subtopic_titles_list = self.request.get('selected_subtopics')
```

Comment:
subtopic_titles_json perhaps? (and change the url param name to include "json" too)

Reply:
I am still stringifying a js array and not a json object, so is 'json' needed in name?

Reply:
Yes because this is a string in json format, right? Only once it stops being in json format and is turned into a list, should you call it a list.

Reply:
Renamed it, used subtopic IDs (which are numbers) and made it a comma separated string.